install/install.sh

```diff
@@ -50,6 +50,11 @@ docker --version || curl -fsSL https://get.docker.com | env -u VERSION sh || (
 )
 systemctl enable docker
 
+#creating a Unix group called docker and adding users to it.
```

Comment:
Please atent to the space after the `#` and also the message, as the following comment explains.